packages/@divvi/mobile/src/i18n/saga.ts

```diff
@@ -39,13 +38,12 @@ export function* handleFetchOtaTranslations() {
         lastFetchTime !== timestamp ||
         DeviceInfo.getVersion() !== lastFetchAppVersion
       ) {
-        const languageMappings = yield* call([otaClient, otaClient.getLanguageMappings])
-        const customMappedLanguage = _.findKey(languageMappings, { locale: currentLanguage })
+        yield* call([otaClient, otaClient.setCurrentLocale], currentLanguage)
 
         const translations = yield* call(
           [otaClient, otaClient.getStringsByLocale],
           undefined,
-          customMappedLanguage || currentLanguage
+          currentLanguage
```

Comment:
Maybe @kathaypacific you wanna confirm this is good. I don't remember the details of the mapping.

Reply:
ah, i wish i added more comments about this. my memory is fuzzy, but i know this was needed for spanish. similar to @bakoushin 's comment below, i think it was needed to handle a difference between declaring spanish as ES vs ES-419 (i think we defined it as ES-419 in the wallet to be exact about which flavour of spanish we were supporting, but in Crowdin we only configured spanish as "ES". i don't remember the reason for why the wallet and Crowdin were implemented differently, but i suspect that we kept it like this for backwards compatibility)

if your app language is set to spanish, we need to call `getStringsByLocale` with "ES" but `currentLanguage` is `es-419` so i don't know if this would work